collections/cams_glc_2017.yaml

```diff
@@ -51,7 +50,7 @@ Extent:
     interval:
       -
         - '2017-01-01T00:00:00Z'
-        - '2017-01-01T00:00:01Z'
+        - '2017-12-31T00:00:00Z'
```

Comment:
I would strongly vote for leaving the old extend here. We updated this for all collections [here](https://github.com/eurodatacube/public-collections/pull/218) because it's quite confusing that we have the whole year as extend but then the collection is only available on the first of January. The underlying problem is a question between what data went in to create the collection/the collection is showing and for what dates does the collection return actually data. Something that we should also document better.

Reply:
Sorry, this file was copied from the one I edited several months ago. I didn't know the changes. Fix to fit the new format.